products/mysql.md

```diff
@@ -33,10 +33,10 @@ releases:
     releaseDate: 2018-04-08
 
 -   releaseCycle: "5.7"
-    latest: '5.7.40'
+    latest: '5.7.41'
     support: 2020-10-31
     eol: 2023-10-31
-    latestReleaseDate: 2022-08-29
+    latestReleaseDate: 2022-12-07
```

Comment:
Note: the date is correct (https://github.com/mysql/mysql-server/releases/tag/mysql-5.7.41) but according to https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-41.html, MySQL 5.7.41 went GA only on 2023-01-17.